src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
```

Comment:
Extending the `Lottery` class here is not necessary and can lead to a misunderstanding of the class hierarchy. The `Application` class should not inherit from `Lottery` unless it is meant to be a specialized type of `Lottery`, which is not the case here.